docs/getting_started.md

```diff
@@ -0,0 +1,54 @@
+# Getting Started with ComposeDB
+
+## Create a schema
+
+* Create a graphQL compatible composite schema, such as
+
+```
+
+
+```
+
+* Compile the schema to [format]
+
+This creates the models on ceramic:
+
+`composedb composite:create [graphql_def_file]`
```

Comment:
Do i have to install compose db globally?

Reply:
Yes. Just install the turborepo